HPXMLtoOpenStudio/resources/utility_bills.rb

```diff
@@ -24,7 +24,7 @@ def self.get_rates_from_eia_data(runner, state_code, fuel_type, fixed_charge, ma
         year_ix = nil
         rows = CSV.read(File.join(File.dirname(__FILE__), '../../ReportUtilityBills/resources/simple_rates/Average_retail_price_of_electricity.csv'))
         rows.each do |row|
-          year_ix = row.index('2021') if row[0] == 'description'
+          year_ix = row.index('2022') if row[0] == 'description'
```

Comment:
It'd be nice if we didn't have the year hardcoded here so that we don't accidentally forget to update this code the next time we update EIA data. Isn't it just the last item in the row?